tasks/unix/install.yml

```diff
@@ -31,6 +22,26 @@
     swarm_client_get_url_python: "{{ python3_path.stdout }}"
   when: python3_path is succeeded
 
+# Fall back to Python2 if Python3 is unavailable
+- name: See if Python2 is installed on the host
+  command: "which python"
+  changed_when: false
+  ignore_errors: true
+  register: python2_path
+  when: python3_path is failed
+
+- name: Use Python2 to download Swarm Client if available
+  set_fact:
+    swarm_client_get_url_python: "{{ python_path.stdout }}"
+  when:
+    - python3_path is failed
```

Comment:
Would it make sense to use a `block` to avoid repeating the `python3_path is failed` condition?

Reply:
Actually, for organizational purposes, I think it might be nice to have both Python3/Python2 tasks in their own blocks. See ccfba72.